web-registry/graphql.schema.json

```diff
@@ -1579,6 +1579,98 @@
         ],
         "possibleTypes": null
       },
+      {
+        "kind": "INPUT_OBJECT",
+        "name": "AddAddrToAccountInput",
+        "description": "All input for the `addAddrToAccount` mutation.",
+        "fields": null,
+        "inputFields": [
+          {
+            "name": "clientMutationId",
+            "description": "An arbitrary string value with no semantic meaning. Will be included in the\npayload verbatim. May be used to track mutations by the client.",
```

Comment:
@wgwz simply note that 4 occurrences appear in schema descriptions with the newline escape sequence `\n`.

Here for example appears `... in the\npayload verbatim.`

Reply:
yeah, i just checked throughout the file and there are dozens of these, don't think it's anything important. should it be there? probably not, but if we removed them i think the codegen would just put them back later.